Mathlib/GroupTheory/Transfer.lean

```diff
@@ -244,3 +244,51 @@ theorem ker_transferSylow_disjoint (Q : Subgroup G) (hQ : IsPGroup p Q) :
 end BurnsideTransfer
 
 end MonoidHom
+
+namespace IsCyclic
+
+open Subgroup
+
+variable {G : Type*} [Group G] [Finite G] {P : Sylow (Nat.card G).minFac G}
```

Comment:
I'm worried that `Sylow (Nat.card G).minFac G` will lead to `motive not type correct` issues down the road.
What do you think of adding a `p : Nat` and the assumption `(Nat.card G).minFac G = p`?